### DISCUSSION TITLE
operator IDs missing geohash

### DISCUSSION THREAD

**Discussion by bamyers99** [transitland/transitland#379] [Q&A] [answered]
https://www.transit.land/operators/o-butte~regional~transit
https://www.transit.land/operators/o-greeley~evans~transit
https://www.transit.land/operators/o-pueblo~co~us
https://www.transit.land/operators/o-tulare~county~regional~transit~agency
https://www.transit.land/operators/o-antelope~valley~transit~authority
https://www.transit.land/operators/o-madison~county~transit~il~us
https://www.transit.land/operators/o-point~or~us
https://www.transit.land/operators/o-altoona~metro~transit
https://www.transit.land/operators/o-rrta~pa
https://www.transit.land/operators/o-cooperative~alliance~for~seacoast~transportation
https://www.transit.land/operators/o-santa~clarita
https://www.transit.land/operators/o-cambria~county~transit~authority
https://www.transit.land/operators/o-high~point~nc~us
https://www.transit.land/operators/o-capitalareatransit~pa
https://www.transit.land/operators/o-memphis~area~transit~authority
https://www.transit.land/operators/o-hartransit~ct~us
https://www.transit.land/operators/o-metra~ga~us
https://www.transit.land/operators/o-hele~on~hi
https://www.transit.land/operators/o-maui~county~transit
https://www.transit.land/operators/o-baton~rouge~cats
https://www.transit.land/operators/o-jefferson~parish
https://www.transit.land/operators/o-lafayette~la
https://www.transit.land/operators/o-acrta~oh~us

**Comment by drewda** [comment, marked as answer]
Hi, in Transitland version 2, the geohash component of a Onestop ID is optional for feeds and operators. (It's still required for stops and routes.) See https://github.com/transitland/transitland-atlas#onestop-ids

**Comment by bamyers99** [comment]
Thanks, docs are out of date.
https://www.transit.land/documentation/onestop-id-scheme/

**Reply by drewda** [reply]
O.K., thanks for noticing.

**Reply by drewda** [reply]
Since the last post here, we have updated the documentation at https://www.transit.land/documentation/onestop-id-scheme/#geohash to describe both the "two part" and "three part" Onestop IDs.
